### PR TITLE
Add Webid to contact card

### PR DESCRIPTION
**Pull request by timbl**
Allows you to add someone's solid id to your vcard data about them. When you do then shows their profile after the contact data.  This in turn has a "chat with me" button and so you contacts become a way of doing stuff.

You can add more than one solidid for those who have more than one.  The profiles are concatenated. Each can be collapsed or expanded under users clicking on an icon in the header.

The styling is not perfect but the functionality is overdue.

### CODE REVIEW

File: contactsPane.js [hunk 1]
```diff
@@ -343,15 +343,15 @@ export default {
...
             .concat(kb.statementsMatching(undefined, undefined, x))
-          var targets = {}
-          ds.map(function (st) {
+          const targets = {}
+          ds.forEach(function (st) {
             targets[st.why.uri] = st
           })
-          var agenda = [] // sets of statements of same dcoument to delete
-          for (var target in targets) {
+          const agenda = [] // sets of statements of same dcoument to delete
```
Comment (TallTed) [inline contactsPane.js:353]:
```suggestion
          const agenda = [] // sets of statements of same document to delete
```

File: contactsPane.js [hunk 2]
```diff
@@ -533,8 +533,8 @@ export default {
                   if (!event.metaKey) {
                     // If only one group has beeen selected show ACL
```
Comment (TallTed) [inline contactsPane.js:534]:
```suggestion
                    // If only one group has been selected, show ACL
```

File: contactsPane.js [hunk 3]
```diff
@@ -729,9 +730,9 @@ export default {
           refreshFilteredPeople(true) // Active: select person if justone left
```
Comment (TallTed) [inline contactsPane.js:730]:
```suggestion
          refreshFilteredPeople(true) // Active: select person if just one left
```

File: groupMembershipControl.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
+
+// Render a control to record the group membetships we have for this agent
```
Comment (TallTed) [inline groupMembershipControl.js:2]:
```suggestion
// Render a control to record the group memberships we have for this agent
```

File: webidControl.js [hunk 5]
```diff
@@ -80,27 +142,110 @@ export async function renderWedidControl (person, context) {
...
+  }
+
+  const h4 = div.appendChild(dom.createElement('h4'))
+  h4.textContent = WEBID_NOUN
+  h4.style = style.formHeadingStyle
+  h4.style.color = style.highlightColor
+
+  const prompt = div.appendChild(dom.createElement('p'))
+  prompt.style = style.commentStyle
+  prompt.textContent = `If you know someones ${WEBID_NOUN} then you can do more stuff with them.
```
Comment (TallTed) [inline webidControl.js:240]:
```suggestion
  prompt.textContent = `If you know someone's ${WEBID_NOUN}, you can do more stuff with them.
```

File: webidControl.js [hunk 6]
```diff
@@ -80,27 +142,110 @@ export async function renderWedidControl (person, context) {
...
+  To record their ${WEBID_NOUN}, click the plus or drag it onto the plus.`
```
Comment (TallTed) [inline webidControl.js:241]:
```suggestion
  To record their ${WEBID_NOUN}, drag it onto the plus, or click the plus 
  to bring up a selector.`
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Minor tweaks to human-focused UI and comment strings

**Review by michielbdejong — APPROVED** [review]

**Pull request merged**